mmyolo/datasets/yolov5_coco.py

```diff
@@ -57,3 +50,13 @@ def prepare_data(self, idx: int) -> Any:
             return self.pipeline(data_info)
         else:
             return super().prepare_data(idx)
+
+
+@DATASETS.register_module()
+class YOLOv5CocoDataset(BatchShapeDataset, CocoDataset):
```

Comment:
The previous `YOLOv5CocoDataset` should give a deprecation warning

Reply:
> The previous `YOLOv5CocoDataset` should give a deprecation warning

I did not add new `YOLOv5CocoDataset`.  I add file `yolov5_datasets.py`.
The plan is to move `YOLOv5CocoDataset`  to  `yolov5_datasets.py` in the next version.